dod-advana/gamechanger-web
# Task/dla s3 source

## Description
Makes documents from the source 'Defense Logistics Agency' able to be opened.

## Tickets 
[UOT-133120](https://jira.di2e.net/browse/UOT-133120)

## Testing Instructions
Has to build on a dev site to be tested because documents can't be opened on local. Search and filter by source 'Defense Logistics Agency'. Open documents from anywhere in the UI that allows you to open documents to ensure they do open.
